packages/venia-ui/lib/components/Routes/routes.js

```diff
@@ -17,7 +18,7 @@ const Routes = () => {
                  * Client-side routes are injected by BabelRouteInjectionPlugin here.
                  * Venia's are defined in packages/venia-ui/lib/targets/venia-ui-intercept.js
                  */}
-                <Route>
+                <AuthRoute>
```

Comment:
I think this makes the whole app require authentication, right?

Reply:
Yes. But if we don't provide the authed prop, it will go straight to a Route component.
Since we inject code with makeRoutesTarget, Routes needs to have the import of AuthRoute.

But I could do something like this so it remains accessible.

```
// Import used in makeRoutesTarget
// eslint-disable-next-line no-unused-vars
import AuthRoute from './authRoute';

const Routes = () => {
    const { pathname } = useLocation();
    useScrollTopOnChange(pathname);

    return (
        <Suspense fallback={fullPageLoadingIndicator}>
            <Switch>
                {/*
                 * Client-side routes are injected by BabelRouteInjectionPlugin here.
                 * Venia's are defined in packages/venia-ui/lib/targets/venia-ui-intercept.js
                 */}
                <Route>```